db_stress_tool/db_stress_listener.h

```diff
@@ -289,7 +289,8 @@ class DbStressListener : public EventListener {
     }
   }
 
-  void OnErrorRecoveryCompleted(Status /* old_bg_error */) override {
+  void OnErrorRecoveryEnd(
```

Comment:
To use the later API

Reply:
Check that recovery is successful?

Reply:
The current stuff we do in this listener will be done even when recovery is not successful.